plugins/coral-plugin-love/client/LoveButton.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {Icon} from 'coral-ui';
+import styles from './styles.css';
+import {I18n} from 'coral-framework';
+import translations from './translations.json';
+import withReaction from 'coral-framework/hocs/withReaction';
+const lang = new I18n(translations);
+
+class LoveButton extends React.Component {
+  handleClick = () => {
+    const {
+      postReaction,
+      deleteReaction,
+      showSignInDialog,
+      reactionSummary
+    } = this.props;
+    const {root: {me}, comment} = this.props;
+
+    // If the current user does not exist, trigger sign in dialog.
+    if (!me) {
+      showSignInDialog();
+      return;
+    }
+
+    // If the current user is banned, do nothing.
+    if (me.status === 'BANNED') {
+      return;
+    }
+
+    if (reactionSummary) {
+      deleteReaction();
+    } else {
+      postReaction();
+    }
+  };
+
+  render() {
+    const {count, reactionSummary} = this.props;
+    return (
+      <button
+        className={`${styles.button} ${reactionSummary ? styles.loved : ''}`}
+        onClick={this.handleClick}
+      >
+        <span>{lang.t(reactionSummary ? 'loved' : 'love')}</span>
```

Comment:
Same comment here as above. This would be more declarative / clear and future proof:

```
<span>{lang.t(alreadyReacted() ? 'loved' : 'love')}</span>
```